test/v2/mocks/services/MockAmendPeriodSummaryService.scala

```diff
@@ -35,8 +34,8 @@ trait MockAmendPeriodSummaryService extends MockFactory {
 
     def amendPeriodSummary(requestData: AmendPeriodSummaryRequest): CallHandler[Future[Either[ErrorWrapper, ResponseWrapper[Unit]]]] = {
```

Comment:
Can replace `Either[ErrorWrapper, ResponseWrapper[Unit]]` with `ServiceOutcome[Unit]`

Reply:
same for all service mocks